_posts/2019-12-20-differential-privacy/differential_privacy.Rmd

```diff
@@ -0,0 +1,470 @@
+---
+title: "Differential Privacy with TensorFlow"
+description: > 
+ Differential Privacy guarantees that results obtained from querying a database are basically independent of the presence in the data of a single individual. Applied to machine learning, we expect that no single training example influences the parameters of the trained model in a substantial way. This post introduces TensorFlow Privacy, a library built on top of TensorFlow, that can be used to train differentially private deep learning models from R.
+ 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydana2019diffpriv
+bibliography: bibliography.bib
+date: 12-20-2019
+categories:
+  - Security
+  - Keras
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/cat.png
+---
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+
+What could be treacherous about summary statistics? 
+
+The famous _cat overweight_ study (X. et al., 2019) showed that as of May 1st, 2019, 32 of 101 domestic cats held in Y., a cozy Bavarian village, were overweight. Even though I'd be curious to determine if my aunt G.'s cat (a happy resident of that village) has been fed too many treats and has accumulated some excess pounds, from the results reported in that study I can't know.
+
+Then, six months later, out comes a new study, ambitious to earn scientific fame. The authors report that of 100 cats living in Y., 50 are striped, 31 are black, and the rest are white; the 31 black ones are all overweight. Now, I happen to know that with one exception, no new cat joined the community, and no cat left. _But_, my aunt moved away to town, to a retirement home, chosen of course for the possibility to bring one's cat.
```

Comment:
```suggestion
Then, six months later, out comes a new study, ambitious to earn scientific fame. The authors report that of 100 cats living in Y., 50 are striped, 31 are black, and the rest are white; the 31 black ones are all overweight. Now, I happen to know that, with one exception, no new cats joined the community, and no cats left. _But_, my aunt moved away from town to a retirement home, chosen of course for the possibility to bring one's cat.
```

Reply:
Not sure why, but we tend to use the plural when speaking of hypothetical quantities. 

Is your aunt in the same town in this scenario? If yes, I'd simplify that last line to:
"_But_, my aunt moved to a retirement home, chosen of course for the possibility to bring one's cat.

Reply:
^^ I now understand that the answer to my question above is no — took me a second. (>ლ) I'm changing "to town" to "from town" since it makes it less ambiguous.

Reply:
oh i see, if the language makes you think there's something wrong... I'll just remove that town thing completely, retirement home should be enough :-)